README.md

```diff
@@ -282,50 +282,50 @@ Global Flags:
 You can step through each of these options interactively by using the `--interactive` flag.
 
 ```
-$ moactl create cluster --interactive
+$ rosa create cluster --interactive
 ```
 
 Otherwise, run the following command to create your cluster with the default cluster settings. The default settings are as follows:
 
 * The AWS region you have configured for the AWS CLI
-* The most recent version of OpenShift available to moactl
+* The most recent version of OpenShift available to rosa
 * A single availability zone
 * Public cluster (Public API)
 * Master nodes: 3
 * Infra nodes: 2
 * Compute nodes: 2 (m5.xlarge instance types)
 
 ```
-$ moactl create cluster --cluster-name=rh-moa-test-cluster
+$ rosa create cluster --cluster-name=rh-rosa-test
 
-I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-moa-test-cluster'
-I: To view list of clusters and their status, run `moactl list clusters`
-I: Cluster 'rh-moa-test-cluster' has been created.
-I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `moactl create idp --help` and `moactl create user --help` for more information.
-I: To determine when your cluster is Ready, run `moactl describe cluster rh-moa-test-cluster`.
+I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-rosa-test'
+I: To view list of clusters and their status, run `rosa list clusters`
```

Comment:
To view **a** list of clusters and their status, run `rosa list clusters`

Reply:
Fixed!